File: cpp-droid/repl.cpp
#include <stdio.h>
#include <string>
#include <cstdlib>
#include <iostream>
#include "engine.cpp"
#include "utils.cpp"

void parse_command(std::string &command) {
    trim(command);
    if (command == ".exit") {
        std::cout << "exiting.. good bye!\n";
        exit(0);
    } else if (command == ".help") {
        std::cout << "Available commands:\n";
        std::cout << ".exit - Exit the program\n";
        std::cout << ".help - Show this help message\n";
    } else {
        execute_sql(command);
    }
}

File: rust-droid/repl.rs
use crate::engine;

pub fn parse_command(command: &str) {
    let command = command.trim();
    if command == ".exit" {
        println!("exiting.. good bye!");
        std::process::exit(0);
    } else if command == ".help" {
        println!("Available commands:");
        println!(".exit - Exit the program");
        println!(".help - Show this help message");
    } else {
        engine::execute_sql(command);
    }
}